.github/workflows/deployment.yaml

```diff
@@ -0,0 +1,83 @@
+name: Deployment
+
+on:
+  push:
+    branches:
+      - master
+      - LISK-1400-Setup-Dshackle
```

Comment:
This branch shouldn't be present in the final config. But let's first test out the CI flow with the version changes suggested below and then after properly testing it, we can remove this branch.
```suggestion
```